tracking/src/trajectory_generation_tracker_node.cpp

```diff
@@ -0,0 +1,25 @@
+#include <tracking/trajectory_generation_tracker.hpp>
+
+using namespace ariitk::tracking;
+
+int main(int argc, char** argv) {
+	ros::init(argc, argv, "trajectory_pub");
+	TrajectoryGenerationTracking tracking(argv);
+
+	ros::Rate loop_rate(tracking.publish_rate_);
+
+	int count = 0;
+	while (count < 10) {
+		ros::spinOnce();
```

Comment:
is this required? can convert into `tracking.waitForStartup()`

Reply:
Yes this is required, to ensure that odometry message has been recieved in tracking.
Will make it a function